config/checkstyle/checkstyle.xml

```diff
@@ -13,7 +13,7 @@
 
         <!-- Imports -->
         <module name="IllegalImportCheck" >
-            <property name="illegalPkgs" value="com.google.common.(?!cache).*,org.apache.commons.text.*,org.slf4j.*"/>
+            <property name="illegalPkgs" value="com.google.common.(?!cache).*,org.apache.commons.text.*,org.slf4j.*,org.jetbrains.*"/>
```

Comment:
Thanks for this. I'll commit this change to the maintenance branches and merge into master so it has a separate audit trail from this pull request.